### PR TITLE
Fix/doc 4288

### PR DESCRIPTION
**Pull request by hghorbel**

### CODE REVIEW

File: docs/Manage-Space.rst [hunk 1]
```diff
@@ -267,12 +267,16 @@ A description of each template is displayed below the selectors.
    -  Close: The user cannot request for joining, but only the space
       manager can invite him.
 
+
+.. note:: Template, Visbility and registration fields have a description of the choice selected to help you taking right decisions when creating a new space.
```
Comment (plamarque) [inline docs/Manage-Space.rst]:
```suggestion
.. note:: Template, Visibility and registration fields have a description of the choice selected to help you choosing the best template for your needs when creating a new space.
```

**Review by plamarque — CHANGES_REQUESTED** [review]

File: docs/Administration.rst [hunk 2]
```diff
@@ -2493,11 +2493,12 @@ All templates are presented in a table with:
 - Visibility status : default visibility option (visible, secret) for the space
 - Registration : default registration policy for the space (open, closed, validation)
 - Applications : list of applications populated by default in the space
-- Banner link : the image that will be used by default as banner for the space (click to display)
 - Permissions : list of groups/roles that are authorized to create a space from this template
+- Banner View : the image that will be used by default as banner for the space (click to display)
 
 .. note:: The Manage Template screen currently only allows to displaying all templates informations. Customising a space template must be done by editing config files as explained in :ref:`developers guide <PLFDevGuide.SpaceTemplates.Extend>` 
-          To display the banner, you can simply click on **Link** so that the banner will be displayed in a new tab.
+          To display the banner, you can simply click on the eye icon so that the banner will be displayed, and you can click outside the banner or on ESC to close it.
```
Comment (mzorai) [inline docs/Administration.rst]:
```suggestion
          To display the banner, simply click on the eye icon. To close it, click outside the banner or on ESC button of the keyboard.
```

**Review by mzorai — CHANGES_REQUESTED** [review]

**Review by mzorai — APPROVED** [review]
LGTM

File: docs/Manage-Space.rst (hunk 1 above)
Comment (plamarque) [inline docs/Manage-Space.rst]:
```suggestion
.. note:: Template, Visibility and Registration fields have a description of the choice selected to help you taking right decisions when creating a new space.
```

Comment (plamarque) [reply docs/Manage-Space.rst]: Also, shouldn't fields be named as they appear on the screenshot to avoid confusion ?

**Review by plamarque — CHANGES_REQUESTED** [review]
typo and a comment about the name of the fiels that don't match the screenshot.

**Comment by hghorbel** [comment]
@plamarque  need your review (ie approve if all is OK)

**Review by plamarque — APPROVED** [review]
LGTM

**Pull request merged**